### DISCUSSION TITLE
Combine AdonisJs InertiaJs and Svelte

### DISCUSSION THREAD

**Discussion by ramabie** [adonisjs/core#2945] [Help]
i am a user of laravel,
after getting information about the adonisjs framework that resembles laravel, finally I tried to learn this adonisjs, and I really admire this framework with its speed.

but when I'm going to do a combination between adonisjs inertiajs with svelte I encounter problems and very little information when I search in the search engine?

how to combine adonisjs, inertiajs and svelte?

Thank you in advance

**Comment by RomainLanz** [comment]
Hey @ramabie! 👋 

What have you tried so far?
Can you share any code examples or even a repository?

**Reply by ramabie** [reply]
I doubt the many shortcomings that I have done.
this is my build combine of adonisJs, inertiaJs dan Svelte https://github.com/ramabie/adonisjs_inertiajs_svelte

**Comment by mubashirjamali101** [comment]
Let me know if I can add svelte also.
